packages/rum-core/src/common/apm-server.js

```diff
@@ -198,7 +198,7 @@ class ApmServer {
     if (logObject.level === 'warn') {
       logObject.level = 'debug'
       this._loggingService.warn(logObject.message)
-    } else {
+    } else if (process.env.NODE_ENV !== 'production') {
```

Comment:
I think the reverse is more readable, i.e. `process.env.NODE_ENV === 'development'`.

besides that also makes sure if they don't supply the `NODE_ENV ` the still get the optimised

Reply:
Not really, Only for production mode debug logs should be removed and must be enabled in dev, test and other sets of possible environments. All of the libraries follow this approach and hence its better to stick to convention.